testdata/baselines/reference/submoduleAccepted/compiler/jsFileMethodOverloads3.errors.txt.diff

```diff
@@ -1,35 +1,26 @@
 --- old.jsFileMethodOverloads3.errors.txt
 +++ new.jsFileMethodOverloads3.errors.txt
-@@= skipped -0, +-1 lines =@@
+@@= skipped -0, +0 lines =@@
 -/a.js(2,5): error TS7012: This overload implicitly returns the type 'any' because it lacks a return type annotation.
 -/a.js(7,5): error TS7012: This overload implicitly returns the type 'any' because it lacks a return type annotation.
--
--
--==== /a.js (2 errors) ====
--    /**
--     * @overload
--        ~~~~~~~~
++/a.js(2,5): error TS7010: 'id', which lacks return-type annotation, implicitly has an 'any' return type.
++/a.js(7,5): error TS7010: 'id', which lacks return-type annotation, implicitly has an 'any' return type.
+ 
+ 
+ ==== /a.js (2 errors) ====
+     /**
+      * @overload
+         ~~~~~~~~
 -!!! error TS7012: This overload implicitly returns the type 'any' because it lacks a return type annotation.
--     * @param {number} x
--     */
--    
--    /**
--     * @overload
--        ~~~~~~~~
++!!! error TS7010: 'id', which lacks return-type annotation, implicitly has an 'any' return type.
```

Comment:
This is just the wrong error message, no? JSDoc overloads are supposed to use the version without a declaration name because they're nameless in the source, right? Even if the synthetic one we make now copies the host function's name. As-is, this error is kinda ambiguous, since nowhere does `id` appear in the overload descriptor.

Reply:
I disagree and collapsed the two intentionally, but if you feel strongly I can change it back.

The reason I like having the name in the error message is that the function name appears a few lines after the tag in the source text. When I read the diff, I feel like the Strada error message withholds useful information on a technicality.

Reply:
As-is, to me, the error feels like it's talking about the host declaration rather than the underlined one, and the span's in the wrong place.

Reply:
If you wanted to compromise, the message `This overload of 'id' lacks return-type annotation and implicitly has an 'any' return type.` would be more apt, I think.